src/main/services/dataCollectors/BuildRequester.groovy

```diff
@@ -9,11 +9,18 @@ import java.text.SimpleDateFormat
 
 import static app.MiningFramework.arguments
 
+
+/**
+ * @requires: that the access key argument is passed and that the project has one of the following build systems:
+ * Maven or Gradle and that the project is a github project, otherwise it will not be executed
+ * @produces: creates a branch with a name following the format: [reduced sha]_build_branch_[timestamp] with a custom travis file
```

Comment:
@rafaelmotaalves reduced sha de que? do merge commit?

o projeto tem que ter um arquivo travis que é usado como base? ou é criado um do zero? deixa isso claro acima

Reply:
@rafaelmotaalves pushes it to the corresponding project on github

o que é o repository's releases page? onde? é provided ou required?

Reply:
O releases page é uma feature do Github que permite armazenar os builds no próprio Github, tentei especificar isso melhor